documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -1731,6 +1731,78 @@ By default, dots in a name are replaced with underscores.
 |===
 
 
+==== Qdrant Sink
```

Comment:
```suggestion
== Qdrant Sink
```
Adding this comment for consistency, because moving forward third level headings and higher are not supported in the downstream documentation. However, I realize that we probably can't implement this suggestion ATM, because, in context, this heading must remain parallel with the other sink topic headings. We'll have to rework this in the future if we productize the server.